File: benchmark/corpus/02_network/c_quiet_fetch.js
// The helper swallows every error and prints "done" whatever happened.
async function warmCache(url) {
  try {
    await (await fetch(url)).text(); // DANGER
  } catch (_e) {
    // ignored
  }
}

const url = (await new Response(Deno.stdin.readable).text()).trim();
await warmCache(url);
console.log("done");

File: benchmark/corpus/02_network/c_quiet_fetch.py
# The helper swallows every error and prints "done" whatever happened.
import sys
import urllib.request


def warm_cache(url):
    try:
        urllib.request.urlopen(url).read()  # DANGER
    except Exception:
        pass


url = sys.stdin.readline().strip()
warm_cache(url)
print("done")
